packages/react-server/src/logger/logger.ts

```diff
@@ -1,50 +1,87 @@
-import {Context} from 'koa';
+/* eslint-disable no-process-env */
+import {Context, Request} from 'koa';
 import chalk from 'chalk';
 import {KoaNextFunction} from '../types';
 
 export const LOGGER = Symbol('logger');
-const PREFIX = chalk`{underline sidecar} ⁓ `;
+const PREFIX = chalk`{underline [React Server]}  `;
 
-interface LoggerOptions {
-  level?: Verbosity;
+export class Logger {
+  private buffer = '';
+  private logger: Pick<Console, 'log' | 'error'> = console;
+
+  log(message: string) {
+    if (process.env.NODE_ENV === 'development') {
+      this.logger.log(`${PREFIX}${message}`);
+    } else {
+      this.buffer = `${this.buffer}\n${message}`;
+    }
+  }
 }
 
-export enum Verbosity {
-  Off,
-  Debug,
+function initialRequestMessage(request: Request): string {
+  const requestMethod = `${request.method.toUpperCase()} "${request.url}"`;
+  return `Started ${requestMethod} for at ${new Date().toISOString()}`;
 }
 
-export class Logger {
-  private logger: Pick<Console, 'log' | 'error'> =
```

Comment:
I'm dropping `console.error`. `.error` prints to `stderr` but in production we `console.log` a lob object with a bunch of metadata. If we have errors we generally look at 500s logs anyways.